src/libs/actions/PaymentMethods.js

```diff
@@ -8,6 +9,28 @@ import Growl from '../Growl';
 import * as Localize from '../Localize';
 import Navigation from '../Navigation/Navigation';
 import * as CardUtils from '../CardUtils';
+import NameValuePair from './NameValuePair';
+
+function deleteDebitCard(fundID) {
+    return API.DeleteFund({fundID})
+        .then((response) => {
+            if (response.jsonCode === 200) {
+                Growl.show('Debit Card successfully deleted!', CONST.GROWL.ERROR, 3000);
+                Onyx.merge(ONYXKEYS.CARD_LIST, {[fundID]: null});
+            } else {
+                Growl.show(Localize.translateLocal('common.genericErrorMessage'), CONST.GROWL.ERROR, 3000);
+            }
+        })
+        .catch(() => {
+            Growl.show(Localize.translateLocal('common.genericErrorMessage'), CONST.GROWL.ERROR, 3000);
+        });
+}
+
+function deletePayPalMe() {
+    NameValuePair.set(CONST.NVP.PAYPAL_ME_ADDRESS, '');
+    Onyx.merge(ONYXKEYS.NVP_PAYPAL_ME_ADDRESS, null);
```

Comment:
Is there any reason we are using `merge()` here? The current guidance is to use `set()`  for simple values.

https://github.com/expensify/react-native-onyx#should-i-use-merge-or-set-or-both